themes/common-theme/webapp/common/css/trumbowygCustom.css

```diff
@@ -0,0 +1,642 @@
+/**
```

Comment:
@bjugl 
Is there any custom styling we have added overriding the default styling of trumbowyg. If not, we could directly use the css files available in ui folder. WDYT?

Reply:
We need the custom css, because Helveticus Themes override the color of at least the Icon that switches to code view and Rainbow Stone does silly things to the outlines of the boxes, due to margins defined on all div containers. 
Those are the two changes I revered two, when I asked about the proper way to overwrite earlier in this PR.

Reply:
I think we should use the default trumbowyg css as is and the changes we want should go in custom css files like we did for Select2
https://github.com/apache/ofbiz-framework/blob/trunk/themes/rainbowstone/webapp/rainbowstone/style.css#L1919

This way we will keep track of custom changes easily. WDYT?